.github/workflows/hygiene.yml

```diff
@@ -68,3 +68,24 @@ jobs:
           && (startsWith(github.event.ref, 'refs/heads/4.')
              || github.event.ref == 'refs/heads/trunk')
           && always()
+
+      - name: Check that labelled/unlabelled .mli files are in sync
+        run: tools/ci/actions/check-labelled-interfaces.sh
+        if: always()
+
+        # If this step fails, it leaves behind a file the presence of which can
```

Comment:
naming the file would help reading the `if:` clauses below

Reply:
Done!